oo7-portal-impl/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "oo7-portal-impl"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+async-std = { version = "1.11", features = [ "attributes" ] }
+futures-channel = "0.3"
+oo7 = { path = "../" }
+ring = "0.17.5"
+secrecy = { version = "0.8", features = [ "alloc" ] }
+serde = {version = "1.0", features = ["derive"]}
+tracing = "0.1"
+tracing-subscriber = "0.3"
+zbus = { git = "https://github.com/dbus2/zbus.git" }
```

Comment:
Does it make sense to mix zbus stable & git?

Reply:
Well, you can switch oo7 to use that and use oo7 from git as well, that would be a good start

Reply:
I think that would be fine... for a separate MR.